altiProfilAdmin/controllers/config.classic.php

```diff
@@ -52,7 +51,20 @@ function index() {
                 $form->setData( $ctrl->ref, $val );
             }
         }
-
+        $tableName =  $form->getData('altiProfileTable');
```

Comment:
I think you should

* quote table name with double-quotes
* add more security by triming content and use a regexp to check the table name, for example `^([a-zA-Z0-9_])+$`

Please check every value coming from the form and either used in a SQL query or displayed in the HTML interface.